training-node/test/users.js

```diff
@@ -138,44 +180,47 @@ describe('users', () => {
         });
     });
   });
-  it('should fail auth because of missing property email', done => {
-    successUserCreate().then(res => {
+  describe('/users GET', () => {
+    it('should success search ', done => {
+      successUserCreate().then(res => {
+        successUserAuth().then(response => {
+          const token = response.headers[tokenManager.HEADER_NAME];
+
+          chai
+            .request(server)
+            .get('/users?offset=0&size=2')
+            .set(tokenManager.HEADER_NAME, token)
+            .then(result => {
+              dictum.chai(result);
+              result.should.have.status(200);
+              result.body.should.have.property('results');
+              result.body.results.should.have.length(1);
+              result.body.should.have.property('total');
+              done();
+            });
+        });
+      });
+    });
+    it('should fail search because is not authenticated', done => {
       chai
         .request(server)
-        .post('/users/sessions')
-        .send({ password: '123' })
+        .get('/users')
         .catch(err => {
-          err.should.have.status(400);
-          err.response.should.be.json;
+          err.response.should.have.status(401);
           err.response.body.should.have.property('error');
           done();
         });
     });
-  });
-  it('should fail auth because is an invalid email', done => {
-    successUserCreate().then(res => {
+    it('should fail search because of bad token', done => {
       chai
         .request(server)
-        .post('/users/sessions')
-        .send({ email: 'martin@walox.com.ar', password: '12346578q' })
+        .get('/users')
+        .set(tokenManager.HEADER_NAME, 'aaaaaaaaaaaaaaaaaaaa')
         .catch(err => {
-          err.should.have.status(400);
-          err.response.should.be.json;
+          err.response.should.have.status(401);
           err.response.body.should.have.property('error');
           done();
         });
     });
   });
-  it('should fail auth because user does not exist', done => {
```

Comment:
Why this test was deleted? because it cant happen?

Reply:
Test still exist !!
https://github.com/wolox-training/mp-express-js/blob/get-users/training-node/test/users.js#L170
Was grouped with others tests of auth